plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-products-controller.php

```diff
@@ -36,6 +36,40 @@ class WC_REST_Products_Controller extends WC_REST_Products_V2_Controller {
 	 */
 	private $search_sku_in_product_lookup_table = '';
 
+	/**
+	 * Suggested product ids.
+	 *
+	 * @var array
+	 */
+	private $suggested_products_ids = array();
+
+	/**
+	 * Register the routes for products.
+	 */
+	public function register_routes() {
+		parent::register_routes();
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/suggested-products',
+			array(
+				'args'   => array(
+					'id' => array(
+						'description' => __( 'Unique identifier for the variable product.', 'woocommerce' ),
+						'type'        => 'integer',
+					),
+				),
```

Comment:
This `id` can be removed right? and maybe replaced with `$this->get_suggested_products_collection_params()`?

Reply:
Or wait, maybe it can be removed as a whole?

Reply:
yeah... checking

Reply:
Removed the whole `args` prop form there here https://github.com/woocommerce/woocommerce/pull/43720/commits/0944caa46698657674874bb1e04f2e56b04407e3